magento/cart2quote/2017-02-01.yaml

```diff
@@ -0,0 +1,13 @@
+
+title:     Remote Code Execution in Qquoteadv/controllers/DownloadController.php
+link:      https://cart2quote.zendesk.com/hc/en-us/articles/115000616303--FIXED-Security-Vulnerability-in-downloadCustomOptionAction
+cve:       ~
+branches:
+    4:
+        time:     2017-02-01 10:45:00
+        versions: ['>=4.1.6', '<5.0'
```

Comment:
How would I best represent this? 
```
    4.1.6:
        time:     2017-02-01 10:45:00
        versions: ['>=4.1.6', '<4.1.6-patch1']
```

Note: Still getting my head around the docs. If this is a foolish question, I'll find the answer shortly.